features/git-town/town.feature

```diff
@@ -1,8 +1,8 @@
 Feature: Show correct git town usage
 
   Scenario: invalid git town command
-    When I run `git town invalidcommand`
-    Then I see
+    When I run `git town invalidcommand` while allowing errors
+    Then I get the error
```

Comment:
I'm okay with this plan. Do you want to leave this as is, and then change it to the new way in your PR?